.github/workflows/check-metabolictasks.yml

```
@@ -8,12 +8,11 @@ jobs:
     timeout-minutes: 60
     strategy:
       matrix:
-        task-type: [essential, verification]
+        task-type: [essential, verification, failure]
```

This error was purposefully introduced so as to trigger a visible error in the workflow that verifies metabolic tasks. This worked as expected for commit [visibly](https://github.com/SysBioChalmers/Human-GEM/pull/635/commits/ba6a1c7c640c407db8f9ffa3ce0698530b6f9e25)
[ba6a1c7](https://github.com/SysBioChalmers/Human-GEM/pull/635/commits/ba6a1c7c640c407db8f9ffa3ce0698530b6f9e25) and will therefore be reverted:
```suggestion
        task-type: [essential, verification]
```